_tips/writing.ja.md

```diff
@@ -0,0 +1,478 @@
+
+<h3><span class="visuallyhidden">Example: </span>見出しの目的</h3>
+
+<p>3、4段落より長い文書では、見出しと小見出しはユーザビリティとアクセシビリティのために重要である。これらの見出しは、読者が文書全体のアウトラインを判断し、関心のある特定の情報にナビゲートするのに役立つ。</p>
+
+<h4><span class="visuallyhidden">例: </span>見出しのレベル</h4>
+
+<p>見出しは1から6までのレベルに分類される。最も高いレベルは「レベル1」で、ページのタイトルや文書の主要なセクションに対応することが多い。</p>
+
+<h3><span class="visuallyhidden">例: </span>意味とフォーマット</h3>
+
+<p>視覚読者は、より大きなサイズのテキストや異なるスタイルのテキストを探すためにページをスキャンすることで、ヘッダーを識別する。支援技術ユーザーはこのような視覚的な変化を見ることができないので、スタイルを変えても十分な手がかりにはならない。</p>
+
+<p>その代わりに、支援技術が見出しを識別できるように、見出しは意味的に「タグ付け」されなければならない。そうすれば、見出しをナビゲーションに使うこともできる。</p>
+
+<p>このため、見出しを追加することは、スクリーン・リーダー・ユーザーにとって最も重要なツールの1つである。タグ付けは通常、視覚的にフォーマットの変更を引き起こすので、多くの文書で調整できることに注意しよう。</p>
+
+<cite>参照 <a href="http://accessibility.psu.edu/headings/">Penn Stateにおける見出しと小見出し</a></cite>
+{% include_cached excol.html type="end" %}
+  </figure>
+</div>
+
+{::nomarkdown}
+{% include_cached box.html type="end" %}
+{:/}
+
+{::nomarkdown}
+{% include_cached box.html type="start" title="詳細情報" class="simple" %}
+{:/}
+
+* **WCAG**
+  * [見出しとラベル 2.4.6](/WAI/WCAG21/quickref/#headings-and-labels) ([理解のため 2.4.6](/WAI/WCAG21/Understanding/headings-and-labels))
+  * [章見出し 2.4.10](/WAI/WCAG21/quickref/#section-headings) ([理解のため 2.4.10](/WAI/WCAG21/Understanding/section-headings))
+  * [情報と関係性 1.3.1](/WAI/WCAG21/quickref/#info-and-relationships) ([理解のため 1.3.1](/WAI/WCAG21/Understanding/info-and-relationships))
+* **ユーザーストーリー**
+  * [   スクリーン・リーダー・ユーザーが見出しを使ってナビゲートする方法](/people-use-web/user-stories/#accountant)
+
+{::nomarkdown}
+{% include_cached box.html type="end" %}
+{:/}
+
+## リンクテキストに意味を持たせる
+
+リンクテキストは､リンク先の内容を説明するように書きましょう｡｢ここをクリック」や「続きを読む」など､あいまいなリンクテキストは使用しないように｡文書の種類やサイズなど､リンク先に関する関連情報を示すこと（例：「提案書類（RTF､20MB）」）｡
+
+
+{::nomarkdown}
+{% include_cached box.html type="start" title="例: リンクテキストを使ってターゲットページを説明する" class="example" %}
+{:/}
+
+<div class="meaningful-links two-column">
+  <figure>
+    <figcaption>{% include_cached icon.html name="ex-circle" label="良くない場合" %} 情報なし</figcaption>
+    <div>
+      <p class="fail">デバイスの独立性についての詳細は, <a href="javascript:return false">ここをクリック</a>.</p>
+    </div>
+  </figure>
+  <figure>
+    <figcaption>{% include_cached icon.html name="check-circle" label="良い場合" %} 意味のある情報</figcaption>
+    <div>
+      <p class="pass"> <a href="javascript: return false">デバイスの独立性に関して</a>.もっと読む</p>
+    </div>
+  </figure>
+</div>
+
+{::nomarkdown}
+{% include_cached box.html type="end" %}
+{:/}
+
+{::nomarkdown}
+{% include_cached box.html type="start" title="詳細情報" class="simple" %}
+{:/}
+
+* **WCAG**
+  * [リンクの目的 (文脈の中で) 2.4.4](/WAI/WCAG21/quickref/#link-purpose-in-context) ([理解のため 2.4.4](/WAI/WCAG21/Understanding/link-purpose-in-context))
+  * [リンクの目的 (リンクのみの場合) 2.4.9](/WAI/WCAG21/quickref/#link-purpose-link-only) ([理解のため 2.4.9](/WAI/WCAG21/Understanding/link-purpose-link-only))
+
+{::nomarkdown}
+{% include_cached box.html type="end" %}
+{:/}
+
+## 画像に代わる意味のあるテキスト
```

Comment:
@jaykishigami My automatic translation may have deceived me again but I want to check with you that the original meaning is preserved. 

Original text is "Write meaningful text alternatives for images", i.e. for every image – except decorative images –, provide a text alternative that is meaningful. 

My translation tool translate this line into something like "Meaningful text instead of images".

What do you think? Do you feel the translation is faithful to the original intent, or could it be improved?

Reply:
## 画像を説明するテキスト､すなわち､装飾のためのイメージは除き､それぞれのイメージを説明する代わりのテキスト

Reply:
@jaykishigami 

> 画像を説明するテキスト､すなわち､装飾のためのイメージは除き､それぞれのイメージを説明する代わりのテキスト

This appears to be a translation of part of my commentary rather than the original text.

What would be your updated translation of "Write meaningful text alternatives for images"?

Thanks!

Reply:
I see. it should be "画像の代わりになる説明文を記述"